app/konnect/mesh-manager/migrate-zone.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Migrate a self-managed zone control plane to {{site.konnect_saas}}
+content_type: tutorial
+---
+
+This guide explains how to migrate a self-managed global control plane to {{site.konnect_saas}}. This can be useful if you manage other services in {{site.konnect_saas}} that way you can view all your entities in one place. 
+
+## Prerequisites
+
+* A universal or Kubernetes cluster with a running zone control plane connected to the self-managed global control plane
+* [The latest version of kumactl](/mesh/latest/production/install-kumactl/) installed and configured to communicate with the self-managed global control plane
+* A [{{site.mesh_product_name}} global control plane in {{site.konnect_short_name}}](/konnect/mesh-manager/service-mesh/#create-a-zone-in-the-global-control-plane)
+* [yq installed](https://github.com/mikefarah/yq)
+
+## Transfer resources from the self-managed global control plane to {{site.konnect_short_name}}
+
+1. Make sure the self-managed global control plane is active:
+  ```bash
+  kumactl config control-planes list
+  ```
+  This should return your self-managed global control plane. You can use port-forward of the On-Prem Global CP API port 5681 to localhost to get quick access with `kumactl`.
```

Comment:
```suggestion
  This should return your self-managed global control plane. If you are using Kubernetes, you can run `kubectl port-forward {resource_name} 5681:localhost` to get quick access with `kumactl`.
```

@lobkovilya I like the idea of keeping the port-forward example. Does the formatting look correct to you? I added the curly brackets because "resource name" is a placeholder. I wasn't sure if there was a port # we could add for localhost or if it would depend on the user's environment.

Reply:
The command looks like this:
```bash
$ kubectl port-forward deployment/{{ site.mesh_cp_name}} -n {{ site.mesh_namespace }} 5681
```